main/conceptual/js-programming/notifiers.md

```diff
@@ -65,10 +65,10 @@ using the `AsyncIterable` API. Each NotifierKit consumer iteration is a lossy sa
 The following properties hold for every sampling subset:
 - Any non-final value from the producer may be missing from the sampling subset.
 - Every non-final value in the sampling subset is a non-final value from the producer
-  (e.g. if "7" is in the subset, then it was in the original producer iteration).
+  (e.g., if "7" is in the subset, then it was in the original producer iteration).
 - Every non-final value in the sampling subset appears in producer order
-  (e.g. if the producer sequence is 1, 3, 8, 5, 9,
-  the sampling subset is guaranteed to not be 8, 1, 5).
+  (e.g., If the producer sequence is 1, 3, 8, 5, 9,
```

Comment:
should "If" be capitalized here?

Reply:
Yes. The "thing" within the parenthesis is a proper sentence, so it should follow the rules of a normal sentence. That's also why I added a period to it inside the parenthesis.

Reply:
The 1st word in the sentence is "e.g." though, yes?

FWIW:
 - [orthography \- When a sentence starts with "e\.g\.", should the e be capitalized? \- English Language & Usage Stack Exchange](https://english.stackexchange.com/questions/28172/when-a-sentence-starts-with-e-g-should-the-e-be-capitalized)